packages/terra-consumer-nav/src/components/quick-links/QuickLinks.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './QuickLink.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: null,
+};
+
+const QuickLinks = ({
```

Comment:
it renders a single <QuickLinks> tag with no content. Nav still functions as expected without the quicklinks (nav-items floats up and keeps UX padding between logo and items)